clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -50,31 +49,29 @@
  */
 @PublicApi
 public final class FileInStream extends InputStream implements BoundedStream, Seekable {
-  /** Logger for this class */
+  /** Logger for this class. */
```

Comment:
I'd prefer reverting the change and setting a standard first before doing this kind of work since it undoubtedly affects more of the codebase.